public/javascript/news-letter-ids.js

```
@@ -0,0 +1,53 @@
+var newsLetterIdMap = {
+  "37" : "Guardian Today UK",
+  "1493" : "Guardian Today US",
+  "1506" : "Guardian Today AUS",
+  "3640" : "Morning Briefing UK",
+  "2636" : "Morning Mail AUS",
+  "217" : "MediaGuardian Briefing",
+  "3698" : "Brexit Briefing",
+  "38" : "Green Light",
+  "3701" : "Lab Notes",
+  "113" : "Poverty Matters",
+  "1866" : "Australian Politics",
+  "218" : "The Fiver",
+  "219" : "The Breakdown",
+  "220" : "The Spin",
+  "3766" : "Guardian Australia Sports",
+  "3745" : "Guardian Documentaries",
+  "3743" : "Weekend Reading",
+  "3890" : "The Long Read",
+  "3322" : "The Long Read",
+  "39" : "Sleeve Notes",
+  "3834" : "Sleeve Notes",
+  "3835" : "Sleeve Notes",
+  "40" : "Close Up",
+  "1950" : "Film Today",
+  "3039" : "Bookmarks",
+  "3866" : "Bookmarks",
+  "3867" : "Bookmarks",
+  "99" : "Art Weekly",
+  "1902" : "Zip File",
+  "3806" : "The Flyer",
+  "2211" : "The Flyer",
+  "3807" : "The Flyer",
+  "1079" : "Money Talks",
+  "105" : "Fashion Statement",
+  "101" : "Crossword Editor's Update",
+  "248" : "The Observer Food Monthly",
+  "3811" : "Best of Guardian Opinion US",
+  "3814" : "Best of Guardian Opinion US",
+  "2313" : "Best of Guardian Opinion US",
+  "3228" : "Best of Guardian Opinion US",
+  "2976" : "Best of Guardian Opinion AUS",
+  "3830" : "The Resistance",
+  "131" : "Book Club",
+  "232" : "Guardian Weekly",
+  "1929" : "Breaking news",
+  "2635" : "First Dog on the Moon",
+  "3888" : "The Recap"
+}
+
+function newsLetterIdToName(id) {
+  return newsLetterIdMap[id] || "Unknown id"
```

maybe throw an Error or alert() something instead of Unknown?

---

this matches previous behaviour